lib/fingerprint.js

```diff
@@ -51,6 +53,21 @@ module.exports = function(opt, hash_provider) {
             return cb();
         }
 
+        // test if the url should be hashed or not
+        if (opt.exclude) {
+            excluded = opt.exclude.some(function (rule) {
```

Comment:
it does, it depends on `orig` from line 59.

Unless you'd like me to move the function to a higher level and bind `orig` to the context.